Tests/ASDisplayNodeTests.mm

```diff
@@ -225,13 +225,6 @@ - (void)testDefaultFirstResponderBehavior {
   XCTAssertFalse([node becomeFirstResponder]);
 }
 
-- (void)testLayerBackedFirstResponderBehavior {
-  ASTestDisplayNode *node = [[ASTestResponderNode alloc] init];
-  node.layerBacked = YES;
-  XCTAssertTrue([node canBecomeFirstResponder]);
-  XCTAssertFalse([node becomeFirstResponder]);
```

Comment:
This test now is invalid because it sets `layerBacked=YES` on a node that has a custom view class. Can't be done!